DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/AdminFieldsViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+
+    public class AdminFieldsViewModel
+    {
+        public AdminFieldsViewModel(CourseCustomPrompts courseCustomPrompts)
+        {
+            AdminFields = courseCustomPrompts.CourseAdminFields;
+        }
+
+        public IEnumerable<CustomPrompt> AdminFields { get; set; }

```

Comment:
If we're setting this up in a similar way to the delegate registration prompts we should probably be using the view models in the same reason unless there is a good reason not to.